crates/bindings-csharp/Runtime/bindings.c

```diff
@@ -208,9 +210,10 @@ int32_t WASI_NAME(fd_write)(__wasi_fd_t fd, const __wasi_ciovec_t* iovs,
     // Note: this will produce ugly broken output, but there's not much we can
     // do about it until we have proper line-buffered WASI writer in the core.
     // It's better than nothing though.
-    _console_log_imp((LogLevel){fd == STDERR_FILENO ? /*WARN*/ 1 : /*INFO*/ 2},
-                     CSTR("wasi"), CSTR(__FILE__), __LINE__, iovs[i].buf,
-                     iovs[i].buf_len);
+    // _console_log_imp((LogLevel){fd == STDERR_FILENO ? /*WARN*/ 1 : /*INFO*/

```

Comment:
Note to self: uncomment this back and verify it works with NAOT (I remember commenting it out because there were some issues).